Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -194,6 +197,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
         style={[
           styles[name],
           styleInterpolator(props),
+          subviewStyle[name],
```

Comment:
@tlvenn Hmm, this is not a pattern we use anywhere. Usually we pass style objects when props are named `*style` instead of a stylesheet object. May be the prop should be named to `subviewStyleSheet`.